iidm/iidm-converter-api/src/test/java/eu/itesla_project/iidm/datasource/ZipMemDataSourceTest.java

```diff
@@ -0,0 +1,40 @@
+package eu.itesla_project.iidm.datasource;
+
+import static org.junit.Assert.*;
+
+import java.io.BufferedOutputStream;
+import java.io.ByteArrayOutputStream;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipOutputStream;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class ZipMemDataSourceTest {
+
+    private byte[] data;
+    
+    @Before
+    public void setUp() throws Exception {
+        ByteArrayOutputStream bao = new ByteArrayOutputStream();
+        try (ZipOutputStream zip = new ZipOutputStream(new BufferedOutputStream(bao));){
+            ZipEntry entry = new ZipEntry("data.xiidm");
+            zip.putNextEntry(entry);
+            zip.closeEntry();
+            zip.close();
+            data = bao.toByteArray();
+        }
```

Comment:
Could you add more than one file in the zip file to ensure data is correctly retrieved